storage/bucket.go

```diff
@@ -1503,6 +1503,19 @@ func toCORSFromProto(rc []*storagepb.Bucket_Cors) []CORS {
 	return out
 }
 
+// Used to handle breaking change in Autogen Storage client OLM Age field
+// from int64 to *int64 gracefully in the manual client
+// Method should be removed once breaking change is made and introduced to this client
+func setAgeCondition(age *int64, ageField interface{}) {
```

Comment:
Can stick with this, but I would probably deal with an int64 and for the pointer use-case write: `c.Set(reflect.ValueOf(&age))`